activities-cancellation-heartbeating/src/activities.ts

```diff
@@ -2,20 +2,21 @@
 import { CancelledFailure, Context } from '@temporalio/activity';
 
 export async function fakeProgress(sleepIntervalMs = 1000): Promise<void> {
+  const { log, info, sleep, heartbeat } = Context.current();
   try {
     // allow for resuming from heartbeat
-    const startingPoint = Context.current().info.heartbeatDetails || 1;
-    console.log('Starting activity at progress:', startingPoint);
+    const startingPoint = info.heartbeatDetails || 1;
+    log.info(`Starting activity at progress: ${startingPoint}`);
```

Comment:
Better use structured logs
```suggestion
    log.info('Starting activity at progress', { startingPoint });
```

Reply:
Good point. Will fix all of these.